src/applications/static-pages/mhv-signin-cta/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import DOMPurify from 'dompurify';
+import { isLoggedIn, isLOA3 } from '~/platform/user/selectors';
+import { signInServiceName } from '~/platform/user/authentication/selectors';
+import UnauthenticatedAlert from './components/messages/UnauthenticatedAlert';
+import UnverifiedAlert from './components/messages/UnverifiedAlert';
+
+/**
+ * MHV Signin CTA widget. This widget displays an alert if the user is not authenticated or verified.
+ * Otherwise, it displays provided HTML content.
+ * @property {HTMLCollection} noAlertContent optional content to display if no alerts are shown
+ * @property {String} signInServiceName the signin service name is available
+ * @property {bool} userIsLoggedIn true if the user is logged in
+ * @property {bool} userIsVerified true if the user is verified
+ */
+export const MhvSigninCallToAction = ({
+  noAlertContent,
+  serviceDescription,
+  serviceName,
+  userIsLoggedIn = false,
+  userIsVerified = false,
+}) => {
+  if (!userIsLoggedIn) {
+    return <UnauthenticatedAlert serviceDescription={serviceDescription} />;
+  }
+
+  if (!userIsVerified) {
+    return (
+      <UnverifiedAlert
+        signInService={serviceName}
+        serviceDescription={serviceDescription}
+      />
+    );
+  }
+
+  // Display the provided children. Note these are HTMLElements and not React.
+  const content = Array.from(noAlertContent).map((child, index) => (
+    <div
+      key={index}
+      // eslint-disable-next-line react/no-danger
+      dangerouslySetInnerHTML={{
+        __html: DOMPurify.sanitize(child.innerHTML),
```

Comment:
Note: There are other examples of using `dangerouslySetInnerHTML` in the repo, but the React community recommends adding `DOMPurify` for security even when we control the static page content.